Makefile

```diff
@@ -41,6 +41,7 @@ bootstrap: toolset
 	mkdir -p $(BOOTSTRAP_PATH) ; \
 	cp Binaries/$(BUILD_CONFIGURATION)/csccore/* $(BOOTSTRAP_PATH) ; \
 	cp Binaries/$(BUILD_CONFIGURATION)/vbccore/* $(BOOTSTRAP_PATH) ; \
+	./build/scripts/crossgen.sh $(BOOTSTRAP_PATH) ; \
```

Comment:
In this case, the two methods should be identical. The only time it matters is in an interactive shell, where you can't execute a script in the current working directory without providing a relative path. Since both of these are valid relative paths, there should be no difference.